### ISSUE TITLE
Infinte back and forth transition between the steps 

### ISSUE THREAD

**Issue by raul1991** [rstaib/jquery-steps#118]
hi

While trying the examples that you have given on your webpage. The one under the advance form example. On step 2 where i choose my age and press next , i get stuck in an infinite loop of back and forth transitions.

Some more details:

Browser : firefox 33.0
OS :  linux/ubuntu 12.04

Is it just that example or the it persists with the plugin itself ?

**Comment by j-a-m-l** [comment]
Me too.

Apart from that, I like the plugin ;).

**Comment by sparksterz** [comment]
I also noticed this one.

**Comment by EnigmaSolved** [comment]
I think it's just that the custom logic for that particular form needs tweaking (eg, `onStepChanging` and `onStepChanged` are probably competing with each other under certain conditions, causing an infinite loop). So that's just a result of the custom code for that form, and not indicative of a problem with the plugin itself (as far as I can tell).

**Comment by sparksterz** [comment]
You're correct, I guess the issue I have is if it's being used in a demo, it'd be nice to have the functionality working well. There's many different ways to solve the issue, it just depends on your usage.

**Comment by EnigmaSolved** [comment]
I agree--it's unfortunate and confusing to have the demo go wonky.
